tools/test/examples/examples_lib.py

```diff
@@ -293,7 +305,7 @@ def compile_repos(config, toolchains):
     return results
 
 
-def update_mbedos_version(config, tag):
+def update_mbedos_version(config, tag, example):
```

Comment:
You've added `example` here but you're not using it at all. Can you add the `continue` step in the loop here as well? The `do_versionning` function will need to be updated in `examples.py` as well.